src/frequenz/sdk/timeseries/_formula_engine/_formula_generators/_pv_power_formula.py

```diff
@@ -17,14 +17,19 @@
 class PVPowerFormula(FormulaGenerator[Power]):
     """Creates a formula engine for calculating the PV power production."""
 
-    def generate(self) -> FormulaEngine[Power]:
+    def generate(  # noqa: DOC502 (ComponentNotFound and RuntimeError are raised
+        #                          indirectly by _get_pv_power_components)
```

Comment:
Feel free to discard this one
```suggestion
    def generate(  # noqa: DOC502 
        # * ComponentNotFound is raised indirectly by _get_pv_power_components
        # * RuntimeError is also raised indirectly by _get_pv_power_components
```